### DISCUSSION TITLE
Text Area for Import CSV / JSON

### DISCUSSION THREAD

**Discussion by proceau** [directus/directus#11274] [Feature Requests]
On import CSV/JSON only from File it's provide, possibile to add "text area" or "paste" alternative source of import

**Comment by jamescammarano** [comment]
There are definitely a lot of features/improvements to do from the CSV/JSON importing.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
